pkg/nfs/nfs.go

```diff
@@ -40,7 +40,7 @@ const (
 )
 
 var (
-	version = "1.0.0-rc2"
+	version = "1.0.0"
```

Comment:
also noticed this while glancing through the code. Probably worth at some point deriving this from git tag/sha in a way that kubectl does?

Reply:
the csi build tools does support injecting git tag into the version: https://github.com/kubernetes-csi/csi-release-tools/blob/master/build.make#L77

We just need to fix this up to use that. We can do that in a followup